runtime/runtime/src/lib.rs

```diff
@@ -1444,7 +1415,7 @@ impl Runtime {
             }
         }
 
-        Ok((gas_used, receipts_to_restore))
+        Ok((0, receipts_to_restore))
```

Comment:
Can you check if this return value is usefully used anywhere? Maybe more code can be removed.

Reply:
This I did remove in a subsequent PR https://github.com/near/nearcore/pull/13181/files